fts-cron/docker-entrypoint.sh

```diff
@@ -14,6 +14,12 @@ then
 elif [[ $RUCIO_VO == 'tutorial' ]]
 then
     j2 /opt/rucio/fts-delegate/renew_fts_proxy_tutorial.sh.j2 > /opt/rucio/fts-delegate/renew_fts_proxy.sh
+elif [[ $RUCIO_VO == 'escape' ]]
+then
+    j2 /opt/rucio/fts-delegate/renew_fts_proxy_escape.sh.j2 > /opt/rucio/fts-delegate/renew_fts_proxy.sh
+elif [[ $RUCIO_VO == 'other' ]]
+then
```

Comment:
this 'other' is redundant, no?

Reply:
It is - on purpose though, as I thought it might be better understandable for users and allow for future changes to the scripts. However, I can remove it, if it's confusing.